### DISCUSSION TITLE
Make video watchable in the overlay panel of a relation.

### DISCUSSION THREAD

**Discussion by ghost** [directus/directus#17161] [Feature Requests]
Would be nice to watch videos through the overlay panel, if file is type `video/*`.
It works in the File Library, but not when setting a relational field in a collection.

**Comment by rijkvanzanten** [comment]
> It works in the File Library, but not when setting a relational field in a collection.

From what I can tell, the `drawer-item` component that renders that overlay panel already includes some logic to render the file preview if a file is present. What relational field are you using where this doesn't work? This might be a bug

**Comment by ghost** [comment]
Here is a video. The first part shows the overlay panel, where I open the file, but no video player appears. The second part of the video shows the same file in the file library, where a video player appears (and only necessary writable meta fields).

https://user-images.githubusercontent.com/2291224/213534229-762d5397-46bf-4fc1-a447-2d9e5dddc6bc.mp4

My relation looks like this …

<img width="353" alt="relation" src="https://user-images.githubusercontent.com/2291224/213535240-d67b95f8-b54b-4147-beb6-f4862da42abf.png">

… where in the projects collection there is a M2A field `slides` with a junction collection `projects_slides` and the two relation collections `project_slide_images` and `project_slide_videos`. The `project_slides_videos` collection contains a file field where I can upload videos.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
